unpackaged/post/config/objects/hed__Education_History__c.object

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <actionOverrides>
+        <actionName>View</actionName>
+        <comment>Action override created by Lightning App Builder during activation.</comment>
+        <content>K12_Education_History_Record_Page</content>
+        <formFactor>Large</formFactor>
+        <skipRecordTypeSelect>false</skipRecordTypeSelect>
+        <type>Flexipage</type>
+    </actionOverrides>
+    <compactLayoutAssignment>K12_Kit_Education_History_Compact_Layout</compactLayoutAssignment>
+    <compactLayouts>
+        <fullName>K12_Kit_Education_History_Compact_Layout</fullName>
+        <fields>Name</fields>
+        <fields>hed__Account__c</fields>
+        <fields>hed__Contact__c</fields>
+        <fields>School_Enrollment_Status__c</fields>
+        <fields>hed__Start_Date__c</fields>
+        <fields>hed__End_Date__c</fields>
+        <label>K12 Kit Education History Compact Layout</label>
+    </compactLayouts>
+    <listViews>
+        <fullName>K12_All_Education_Histories</fullName>
+        <columns>NAME</columns>
+        <columns>hed__Account__c</columns>
+        <columns>hed__Contact__c</columns>
+        <columns>hed__Start_Date__c</columns>
+        <columns>Start_Reason__c</columns>
+        <columns>hed__End_Date__c</columns>
+        <columns>End_Reason__c</columns>
```

Comment:
@m-kinkade Start Reason and End Reason are fields managed by K-12 kit and deserving of namespace tokens.